### ISSUE TITLE
Erroneous "Pairing failed" notifications. 

### ISSUE THREAD

**Issue by egingell** [RikkaApps/Shizuku#380]
Please report bugs of Shizuku itself.

**Requirements:**

- [x] Shizuku version is up-to-date
- [x] Shizuku is downloaded from official channels (GitHub release or Google Play)
- [x] Shizuku is not running in a virtual environment or broken ROM (GrapheneOS)
- [x] (Root users) No Xposed installed / Xposed is not enabled for Shizuku

Change "[ ]" to "[x]" if it meets the requirements.

**Reports not meet the requirements will be immediately closed**.

**Information:**

- Shizuku version: 13.5
- Mode: [adb, root] adb
- Android version: 13
- Device: (optional) Samsung Galaxy A32 5G

**Logs:**

Related **full logcat**.

**Describe the bug:**

I have Shizuku paired successfully,  but occasionally I'll get a notification that said "pairing failed". Then when go to Shizuku, it says it's running (or it starts fine). Then if I stop and restart it, it either starts fine or I just have to turn Wireless Debugging back on and it still starts fine without needing to repair.

Long story short, I get an error notification despite there being no error.

**Additional context:**

**Comment by mvevitsis** [comment]
Did you ever solve this?

**Comment by panyann** [comment]
So I guess the "solution" is to disable notifications? xD

**Comment by egingell** [comment]
That was my solution. Why? Did you fix it?

**Comment by panyann** [comment]
@egingell Yeah, I was being sarcastic...

**Comment by egingell** [comment]
@panyann Honestly, though, I have a different phone with Android 14. I don't recall disabling the notification on this phone.

**Comment by vvb2060** [comment]
fixed 2d56a5a 4d88dbe
